### ISSUE TITLE
Discrepancy in energy shares value 

### ISSUE THREAD

**Issue by gerard76** [jobvk/Home-Assistant-Windcentrale#59]
### Checklist

- [X] I'm running the [latest](https://github.com/jobvk/Home-Assistant-Windcentrale/releases/latest) version of the Windcentrale integration.
- [X] I checked if there is not a duplicate issue with the same question.
- [X] I have read the [README.md](https://github.com/jobvk/Home-Assistant-Windcentrale/blob/master/README.md) page.
- [X] This issue is a bug related to the Windcentrale integration.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] I have filled out the issue template to the best of my ability.

### The problem

I have shares in a wind turbine that I own since august this year and it shows  1503kWh as `energy_shares` in the integration, but in reality it is 360kWh.

`month_shares` show the correct number

<img width="731" alt="Screenshot 2022-11-06 at 09 31 56" src="https://user-images.githubusercontent.com/765547/200161428-cd1935c3-4711-4232-beb0-e4fdfc7925e6.png">
<img width="986" alt="Screenshot 2022-11-06 at 09 31 41" src="https://user-images.githubusercontent.com/765547/200161433-53eb7bae-b323-4abf-911f-37af7a105467.png">

### What is version of Home Assistant Core did you recieve the bug?

 2022.11.1

### What is version of Home Assistant Supervisor did you recieve the bug?

2022.10.2

### What is version of Home Assistant Supervisor?

2022.10.2

### What is version of the Windcentrale integration?

0.2.5

**Comment by jobvk** [comment]
Yes that's because that sensor uses the live data API. In doing so, it looks at what the year energy total is with the year_production value. That was when you took the screenshot 2085556 kWh.

Assuming you have 4 shares of the Vier Winden the formula would look like this:
`(2085556  / 5512) * 4 = 1513.4658926`

And since you only bought your shares in August, the value is indeed not correct. Unfortunately, I cannot show the data from month_shares as that APi updates only once the day after.

**Comment by gerard76** [comment]
`wontfix` then.

Maybe slight change in the README description of `sensor.name_production_year_shares` and `sensor.name_energy_shares` which are now basically equal with:
```
The energy that your shares of the wind turbine has produced this year.
```
and
```
The energy produced by your shares of the wind turbine this year.
```

But I cannot think of a succinct way to convey the actual meaning
